library/src/main/java/com/google/android/material/motion/runtime/Performer.java

```diff
@@ -50,6 +50,27 @@
   }
 
   /**
+   * {@link Performer}s can implement this interface to understand when named plans are added to
+   * or removed from this performer.
+   */
+  public interface NamedPlanPerformance {
+
+    /**
+     * Executed when a {@link NamedPlan} is added to this Performer.
```

Comment:
for consistency, please start this method javadoc with a present-tense verb. Also see `PlanPerformance#addPlan` for structure.

"Provides a {@link NamedPlan} to this Performer. The Performer is expected to first remove the effects of any prior plan with the same name, then execute the new plan."

Same for method below.